src/bin/intern.ts

```diff
@@ -25,6 +25,18 @@ getConfig()
 
       intern.configure({ reporters: 'runner' });
       intern.configure(config);
+
+      if (
+        (intern.environment === 'browser' &&
+          (intern.config.suites &&
+            intern.config.suites.some(pattern => pattern.endsWith('.ts')))) ||
+        (intern.config.plugins &&
```

Comment:
I think a paren may be off here. This is saying `if (browser || plugins) { ... }`, but plugins in node should be OK.

Reply:
Whoops, added more test cases to better verify this behavior and fixed those parens.